packages/react/package.json

```diff
@@ -22,7 +22,7 @@
     "@babel/preset-env": "^7.25.4",
     "@babel/preset-react": "^7.24.7",
     "@babel/preset-typescript": "^7.24.7",
-    "@iframe-resizer/child": "^5.2.6",
+    "@iframe-resizer/child": "^5.3.1",
```

Comment:
En dat is ook gedaan, maar blijkbaar ben ik in deze `package.json` vergeten de oude weg te halen.